tests/testthat/test-renderer1-text.R

```diff
@@ -30,3 +30,52 @@ test_that("text may contain commas and parentheses", {
   expect_that(any(grepl("\\(", txt)), is_true())
   expect_that(any(grepl(",", txt)), is_true())
 })
+
+### Test text rotation option
+library(data.table)
```

Comment:
can you please use data.frame instead of data.table, since we currently do not have any dependency on data.table? (this should cause the CRAN test to fail but we are not seeing it because it fails on the version string before it gets to that check)